art/estimators/certification/randomized_smoothing/randomized_smoothing_numpy.py

```diff
@@ -0,0 +1,148 @@
+# MIT License
```

Comment:
Please rename this file to `numpy.py`

Reply:
I'm confused. You want me to rename `randomized_smoothing_numpy.py` to `numpy.py`?

Reply:
Yes, that's what I thought, because it would be following the naming pattern of the other estimator modules where each file has just the name of the supported framework.